mutt/date.c

```diff
@@ -56,6 +57,19 @@ static const char *const Months[] = {
   "Jul", "Aug", "Sep", "Oct", "Nov", "Dec",
 };
 
+/**
+ * Special cases for humanized timestamps
+ */
+static const unsigned ONE_HOUR = 3600;          // In terms of seconds
+static const unsigned ONE_DAY = 24 * ONE_HOUR;  // Again, expressed in seconds
+static const char *const E0h = "just now";      // Within the hour
+static const char *const E1h = "an hour ago";   // Between an hour and two hours
+static const char *const E2h = "few hours ago"; // Between two and four hours
+static const char *const E6h = "earlier today"; // Over four hours ago, the same day
+static const char *const E1d = "yesterday";     // Anytime the previous day
+static const char *const ENd = "days ago";      // N days ago; be precise
```

Comment:
Please can you change the comments to `///<`
This string is magic to Doxygen, meaning: This comment applies to the thing before it.

See: https://neomutt.org/code/